deployment/docker/rockylinux/start_gunicorn_serch_engine.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+NAME="omero_search_engine"
+USER root
+PYTHONPATH=~/.pyenv/versions/searchengine/bin/
```

Comment:
should we go for a location similar to what we have for omero e.g.
``/opt/omero/searchengine/venv3``? instead of homedir

Reply:
Sorry, this line was redundant. I have cleaned up the script.